### PR TITLE
chore(2170): console update ledger entries columns

### LINKED ISSUE #2170: [console]: Update ledger entries columns
The ledger entry API is changing

## Tasks

- [ ] Update ledger entry table columns to accommodate renamed query fields
- [ ] Update filter arguments to accommodate renamed inputs
- [ ] Adjust table. 
  - [ ] Add Sender and Receiver columns
  - [ ] Conditionally render truncated partyId or market code (with block explorer links) along with account type in each cell. EG `0x123...345 General`, `TSLA.QM21 Margin`

## Additional information

See backend PR: https://github.com/vegaprotocol/vega/pull/6834

### PR DESCRIPTION
**Pull request by macqbat**
# Related issues 🔗

Closes #2170

# Description ℹ️

Adjust columns of ledger entries according to API changes. To merge after API release

# Demo 📺

image

image

# Technical 👨‍🔧

Add new columns Sender and Receiver for display more data about ledger flow

### CODE REVIEW

**Review by candida-d — COMMENTED** [review]
Left a couple of comments, probably in the wrong place! Also, in the screnshot it looks like sender and receiver are the wrong way around, but I believe it's a core bug: https://github.com/vegaprotocol/vega/issues/6934

File: libs/ledger/src/lib/ledger-table.tsx [hunk 1]
```diff
@@ -27,11 +31,53 @@ export const TransferTooltipCellComponent = ({
...
+};
+
+const LedgerCellRenderer = ({
+  accountType,
+  partyId,
+  marketName,
+}: LedgerCellRendererProps) => {
+  return (
+    <div className="flex flex-col justify-around leading-5 h-full">
+      <div className="flex">{`${t('Account')}: ${truncateByChars(
```
[the remark this answers is not in the document]
Comment (macqbat) [reply libs/ledger/src/lib/ledger-table.tsx:47]: I removed the labels and put them to the title of the element

**Review by MadalinaRaicu — APPROVED** [review]
Perhaps we can change `Account` label before party ID to just `Party`

File: apps/trading-e2e/src/support/mocks/generate-ledger-entries.ts [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+        },
+        __typename: 'AggregatedLedgerEntriesEdge',
+      },
+      {
+        node: {
+          vegaTime: '1669209347054198000',
+          quantity: '0',
+          assetId: 'asset-id',
+          transferType: Types.TransferType.TRANSFER_TYPE_MARGIN_HIGH,
+          receiverAccountType: Types.AccountType.ACCOUNT_TYPE_EXTERNAL,
```
[the remark this answers is not in the document]
Comment (macqbat) [reply apps/trading-e2e/src/support/mocks/generate-ledger-entries.ts:47]: not sure about this requirement. But there will be some next works at the ledger entries list. I thin we can address this thing at those tasks

**Pull request merged**